doc/js_tutorials/js_setup/js_nodejs/js_nodejs.markdown

```diff
@@ -333,13 +358,15 @@ function installDOM(){
 ### Execute it ###
 
 -   Save the file as `exampleNodeCanvasData.js`.
--   Make sure the files `aarcascade_frontalface_default.xml` and `haarcascade_eye.xml` are present in project's directory. They can be obtained from [OpenCV sources](https://github.com/opencv/opencv/tree/5.x/data/haarcascades).
+-   Make sure the files `face_detection_yunet_2023mar.onnx` are present in project's directory. They can be obtained from [OpenCV Model Zoo](https://github.com/opencv/opencv_zoo/tree/main/models/face_detection_yunet).
```

Comment:
I propose to add download step as in https://github.com/opencv/opencv/pull/25468 and drop the dependency in the description.

Reply:
Done.